test/test_connectionpool.py

```diff
@@ -223,13 +223,19 @@ def test_max_connections(self):
 
             assert pool.num_connections == 1
 
-    def test_pool_edgecases(self):
+    def test_pool_edgecases(self):  # TODO rename this
```

Comment:
I think this case is meant to test that if you add 2 conns to a pool with pool.maxsize=1, the 2nd connection will get closed instead of getting put onto the queue.
Is that correct? If so, I'll rename the test case.

Reply:
Yeah you can rename this test, thanks